src/tersets.zig

```diff
@@ -39,6 +39,9 @@ pub const Method = enum {
 /// A point represented by `time` and `value`.
 pub const Point = struct { time: usize, value: f64 };
 
+/// Segment represented by the `start_point` and `end_point`.
```

Comment:
Maybe state what a segment is before stating what it is represented by. I do not think it is as self explanatory as point.

Reply:
This is tricky. Finding a proper definition of a Segment without creating confusion is not straightforward. 
Turns out that the proper definition is:  Part of a line that is bounded by two distinct endpoints.
However, that inserts the concept line, which can be problematic. I propose the following:
/// This structure models a straight line segment from `start_point` to `end_point`.